META-INF/MANIFEST.MF

```diff
@@ -15,5 +15,6 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.lsp4j,
  org.eclipse.lsp4j.jsonrpc
 Automatic-Module-Name: LSP_CheckerFramework
-Bundle-RequiredExecutionEnvironment: JavaSE-12
+Bundle-RequiredExecutionEnvironment: JavaSE-12,
```

Comment:
Can you please verify that removing the `JavaSE-12,` here works?

Reply:
Yes, it works fine for me. I remove the JavaSE-12 in the pr.